### PR TITLE
Peripheral Cluster defaults

### PR DESCRIPTION
**Pull request by burrbull**
r? @Disasm 

Depends on https://github.com/japaric/svd/pull/74

Instead of #371

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -39,7 +39,7 @@ error-chain = "0.11.0"
 inflections = "1.1.0"
 log = { version = "~0.4", features = ["std"] }
 quote = "0.3.15"
-svd-parser = "0.7"
+svd-parser = {git = "https://github.com/burrbull/svd", branch = "defs"}
```
Comment (therealprof) [inline Cargo.toml]: That's a big nono...

**Comment by burrbull** [comment]
It's only for testing.

**Comment by therealprof** [comment]
Please mark such PRs with a do not merge label then to prevent an accidental merge.

File: src/generate/peripheral.rs [hunk 2]
```diff
@@ -163,13 +163,15 @@ pub fn render(
         return Ok(out);
     }
 
+    let defaults = p.defaults.derive_from(defaults);
```
Comment (Disasm) [inline src/generate/peripheral.rs]: I'd rather use `register_properties` or `default_register_properties` here to express the meaning of this "default". The same for the struct fields. This property set is called `registerPropertiesGroup` in SVD spec.

**Comment by Disasm** [comment]
Thanks! Apart from WIP nature of this PR and naming issues, everything else LGTM. Please ping me when `svd` PR lands.

**Comment by burrbull** [comment]
bors try

**Comment by burrbull** [comment]
`msp430g2553` contains a lot of empty `enumeratedValues`.
What should we do with this?
cc @therealprof 
cc @Disasm

**Comment by burrbull** [comment]
cc @pftbest

**Comment by burrbull** [comment]
`sed -i 's+<enumeratedValues/>++g' msp430g2553.svd`

**Comment by pftbest** [comment]
enumeratedValue are marked as `1..*` in the spec, so enumeratedValues must not be empty. I will fix msp430g2553.svd

**Comment by burrbull** [comment]
@pftbest I still see a lot of `<enumeratedValues/>` in
https://raw.githubusercontent.com/pftbest/msp430g2553/v0.1.3/msp430g2553.svd

**Comment by pftbest** [comment]
@burrbull your link has incorrect tag, please change the tag to `v0.1.3-svd`

**Comment by burrbull** [comment]
bors try

**Comment by burrbull** [comment]
:man_facepalming:

**Comment by burrbull** [comment]
Done.

**Pull request merged**